### DISCUSSION TITLE
How can I limit the teleport field ???

### DISCUSSION THREAD

**Discussion by kudomayuko** [mozilla/hubs#5118] [Spoke Questions] [answered]
Hey there, 

this is my first time to ask here about the technical question..
I am super beginner to build the world in Hubs.
So I am facing a problem, maybe could you help me ..? That would be nice....

I wanted to limit the field of activity, and made walls around the space.
It did work, when you control with keyboard.(q/w/e/a/s/d)
However, when you want to teleport (like, right long-click of mouse) you can move easily out of the field.

How can I fix it?
Here is my world.
https://hubs.mozilla.com/59aE2up/sleepy-simple-outing

Thanks!

**Comment by matthewbcool** [comment, marked as answer]
Such a fun world! Users will be able to teleport anywhere there is nav-mesh.  The nav-mesh allows the avatar to move throughout the space and dictates where they can go. Right now, it looks like your nav mesh extends further than you would like. If you disable the other objects in the room and generate the nav mesh when you have your floor plan, you can create a more specific nav mesh. 

Here I'm disabling objects and regenerating the floor plan- watch what happens: https://gyazo.com/3bc41806145ec6e796914de928617765

Welcome to join us in discord: https://discord.gg/sBMqSjCndj
We also have the hubs community meetup weekly 11:30AM Pacific in the hubs discord #meetup channel where you can ask questions live.

**Reply by kudomayuko** [reply]
Hello Matthew!!
Thank you very very much for your Tipp!!
I was saved by you from the outrageous borderless world.

Thanks a lot!
All the best,
mayuko
